unstructured/ingest/main.py

```diff
@@ -14,14 +15,17 @@
     SimpleGoogleDriveConfig,
 )
 from unstructured.ingest.connector.reddit import RedditConnector, SimpleRedditConfig
-from unstructured.ingest.connector.s3_connector import S3Connector, SimpleS3Config
+from unstructured.ingest.connector.s3 import S3Connector, SimpleS3Config
 from unstructured.ingest.connector.wikipedia import (
     SimpleWikipediaConfig,
     WikipediaConnector,
 )
 from unstructured.ingest.doc_processor.generalized import initialize, process_document
 from unstructured.ingest.logger import ingest_log_streaming_init, logger
 
+with suppress(RuntimeError):
+    mp.set_start_method("spawn")
```

Comment:
Big change here and I agree it seems to be a good one.

To confirm, I tested all the other example scripts to confirm on both linux and mac. 😅